lib/widgets/screens/profile/profile.dart

```diff
@@ -26,52 +119,198 @@ class ProfileScreenScroll extends StatefulWidget {
+                website: company.websites?.first.value,
+                stage: company.companyStage?.translatedValue,
+                city: "Washington D.C.", //TODO
+                country: "USA", //TODO
+                industry: company.industries?.first,
+                expertisesSought: widget.userData.groupMemberships
+                    .firstWhere((g) => g.groupIdent == GroupIdent.mentees)
+                    .maybeWhen(
+                        menteesGroupMembership: (g) => g.soughtExpertises
+                            .map((e) => e.translatedValue!)
+                            .toList(),
+                        orElse: () => []),
+                mission: company.description,
+                imageUrls: [
+                  "https://st2.depositphotos.com/1326558/7163/i/600/depositphotos_71632883-stock-photo-mexican-tacos-with-meat-vegetables.jpg",
+                  "https://st3.depositphotos.com/13349494/32631/i/600/depositphotos_326317470-stock-photo-cropped-view-man-adding-minced.jpg",
+                ],
+                motivation:
+                    'Sit amet justo donec enim diam vulputate ut pharetra sit amet aliquam id diam maecenas ultricies. Mi eget mauris pharetra et ultrices neque ornare aenean euismod elementum nisi quis eleifend.', //TODO
+              ),
+            ),
+          ],
+          if (widget.userData.offersHelp) ...[
+            const Divider(),
+            HowCanIHelpSection(
+              expertises: widget.userData.groupMemberships
+                  .firstWhere((g) => g.groupIdent == GroupIdent.mentors)
+                  .maybeWhen(
+                    mentorsGroupMembership: (g) =>
+                        g.expertises.map((e) => e.translatedValue!).toList(),
+                    orElse: () => [],
+                  ),
+              industries: widget.userData.groupMemberships
+                  .firstWhere((g) => g.groupIdent == GroupIdent.mentors)
+                  .maybeWhen(
+                    mentorsGroupMembership: (g) =>
+                        g.industries.map((e) => e.translatedValue!).toList(),
+                    orElse: () => [],
+                  ),
+              mentoringPreferences: const [
+                "Weekly check-ins",
+                "Monthly check-ins",
+                "Informal chats",
+                "Formal meetings",
+                "Spot mentoring",
+              ], //TODO
+              expectations:
+                  'Sit amet justo donec enim diam vulputate ut pharetra sit amet aliquam id diam maecenas ultricies. Mi eget mauris pharetra et ultrices neque ornare aenean euismod elementum nisi quis eleifend.', //TODO
             ),
-            const ProfileAboutMe(),
-            const ExperienceAndEducation(),
-            const AboutMyBusiness(),
-            const HowCanIHelpSection(),
           ],
-        ),
+          const Divider(),
+          ExperienceAndEducation(
+            experience: [
+              ExperienceInput(
```

Comment:
this is referred to as `BusinessExperience` in the backend, provided in #135

Reply:
Fields:
`BusinessExperience.jobTitle` = `position` 
`BusinessExperience.businessName` = `companyName` 
`BusinessExperience.startDate`
`BusinessExperience.endDate` (optional)
`BusinessExperience.city`
`BusinessExperience.state`
`BusinessExperience.country`
^ maybe it makes more sense to combine the above into one `location` field, but I kept it consistent. Other location fields in the backend are all separating city/state/country. Once we know how these fields are defined in the onboarding/edit profile forms, we could consider combining these fields.